llvm/include/llvm/CASObjectFormats/ObjectFormatHelpers.h

```diff
@@ -51,6 +51,11 @@ bool compareSymbolsBySemanticsAnd(
 bool compareSymbolsByLinkageAndSemantics(const jitlink::Symbol *LHS,
                                          const jitlink::Symbol *RHS);
 
+bool compareSymbolsByAddressAnd(
```

Comment:
Do you want to change the name of the function to something like
`compareSymbolsByAddressAndCompareFunction` Just leaving it the way it is doesn't sound good to me.

Reply:
This is intentionally named in the same convention as other functions here.